### PR TITLE
Update BIDSMapping and XnatToBids

### PR DESCRIPTION
**Pull request by praitayini**
Added features: 
1) BIDSMapping/XnatToBids - Added run_number mapping 
2) BIDSMapping                    - Updated --create_default to get the default from project ID (rather than LANDMAN project) for scan_type or series_description
3) BIDSMapping                   - Updated the --template to get scan series_descriptions as well
4) XnatToBids                       - Modified dataset_description_file function in order to use it in job_template for BIDS-App.

### CODE REVIEW

**Comment by baxpr** [comment]
Nice! @praitayini can you summarize what testing you've done?

**Comment by praitayini** [comment]
> Nice! @praitayini can you summarize what testing you've done?

1. I ran --create, --update, --replace for --type run_number in BIDSMapping. Then, downloaded the data using Xnatdownload bids. I also tested if there are error in csv file for example run_no instead or run_number
2. I ran --create_default for LANDMAN_UPGRAD using --xnatinfo scan_type and —xnatinfo series_description. Then, checked the mapping at project level.
3. I ran --template for --xnatinfo series_description and --xnatinfo scan_type.
4. I ran few sessions on ZALD_TTS using dax build/launch/update for BIDS-App mriqc which requires dataset_description file.

Please let me know if you have more questions/concerns.

**Comment by praitayini** [comment]
@baxpr @bud42 Can we merge this request?

**Comment by baxpr** [comment]
I'll defer to @bud42
